packages/webapp/src/inductions/api/eden-contract.ts

```diff
@@ -16,7 +16,8 @@ const INDEX_BY_ENDORSER = 2;
 const INDEX_BY_INDUCTION = 3;
 
 export const getInduction = async (
```

Comment:
I love the way you decoupled the `getInduction` from `getEndorsementsByInductionId`! I'd just go further and not add the latest boolean flag on the `getInduction` function, we usually avoid adding boolean parameters to functions because it make it harder to read... eg: when you see this `getInduction(1234, true)` you don't know what this true stands for.

So I'd suggest us to go further and do this:
```ts
// function without boolean flag, only lookups for the induction
getInduction(1234)
// the new endorsements lookup: perfect
getEndorsementsByInductionId(1234)

// new function, this will call both above functions like the way you wrote with the boolean flag
getInductionWithEndorsements(1234) 
```

Another common approach is to have an optional object for options, if we think above function will grow and lookup for endorsements, another related members, or whatsoever:
```ts
getInduction(1234, { withEndorsements: true, withAnotherThing: true})
```

Ah, anyway, I love the refactor, these are just suggestions! :)

Reply:
Good suggestion. This was me trying to keep the PR changes minimal and adding new required attributes or changing the call signature too much meant touching additional files. But I think you're right; in this case, it's better to go ahead and do it. 👍